android/app/build.gradle

```diff
@@ -108,6 +108,15 @@ android {
             signingConfig signingConfigs.release
         }
     }
+    configurations.all {
+    resolutionStrategy {
+        eachDependency {
+            if ((requested.group == "org.jetbrains.kotlin") && (requested.name.startsWith("kotlin-stdlib"))) {
+                useVersion("1.8.0")
+            }
+        }
```

Comment:
What's all this for?

Reply:
I was facing an error while running the app. I'm attaching a link for a better explanation.

1. https://github.com/flutter/flutter/issues/119247
2. https://developer.android.com/build/dependencies#resolution_errors

Reply:
Ok thanks for the info!